SecureSubmit.php

```diff
@@ -568,22 +568,21 @@ function report_export(){
                 'additional_info4','additional_info5','additional_info6','additional_info7',
                 'additional_info8','additional_info9','additional_info10');
 
-
-            $transactions = $wpdb->get_results('select * from '.$table_name.' order by id desc;' , 'ARRAY_A');
+            $transactions = $wpdb->get_results( $wpdb->prepare("SELECT * FROM %s order by id desc",$table_name ) , 'ARRAY_A' );
 
             $headers = array();
             foreach ( $fields as $key => $field ) {
                 $headers[] = '"' . strtolower( $field ) . '"';
             }
-            echo implode( ',', $headers ) . "\n";
+            echo implode( ',', esc_html($headers) ) . "\n";
```

Comment:
Can we modify this as follows:

```
            foreach ( $fields as $key => $field ) {
                $headers[] = '"' . esc_html( strtolower( $field ) ). '"';
            }
            echo implode( ',', $headers ) . "\n";
```

...because if we move esc_html() then we no longer have to rely on type juggling within the implode() call